client/my-sites/site-settings/settings-controller.js

```diff
@@ -58,7 +61,7 @@ export function siteSettings( context, next ) {
 	}
 	recordPageView( basePath + '/:site', analyticsPageTitle );
 
-	next();
+	redirectIfDuplicatedView( 'options-general.php' )( context, next );
```

Comment:
Needed to happen after the above redirect checks so running the extra middleware here instead of in the controller like all the others.